project/plugins.sbt

```diff
@@ -3,7 +3,7 @@ addSbtPlugin("com.47deg"          % "sbt-microsites"        % "0.7.17")
 addSbtPlugin("com.jsuereth"       % "sbt-pgp"               % "1.1.0")
 addSbtPlugin("com.github.gseitz"  % "sbt-release"           % "1.0.7")
 addSbtPlugin("com.lucidchart"     % "sbt-scalafmt-coursier" % "1.15")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"           % "0.6.22")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"           % "0.6.24")
```

Comment:
TODO: this causes the plugin to emit some deprecation warnings during the build. Will sort that out in a separate PR.